### ISSUE TITLE
Version 2.2.0 Not Passing questions

### ISSUE THREAD

**Issue by digital420** [cleartonic/twitchtriviabot#22]
Was advised to use older version due to a bug with the questions. This version 2.2.0 is not reading the triviaset.csv instead it does the following output which is directly from the channel

Trivia has begun! Question Count: 1. Trivia will start in 5 seconds.
Question 1: category: Question
a__w__
_nsw_r
Question being skipped. The answer was ** answer **.

triviaset.csv
cat triviaset.csv 
category,question,answer,answer2,creator
Lame,Does this work?,yes,Yes,

**Comment by digital420** [comment]
Log outpout 
09/11/2023 01:49:22 PM Loading trivia data...
09/11/2023 01:49:22 PM Creating session set data. Population 2, session 1
09/11/2023 01:49:22 PM Finished setting up Session.
09/11/2023 01:49:22 PM Active
09/11/2023 01:49:27 PM Category: category
Question: Question
Answers:  ['answer', 'answer2']
09/11/2023 01:50:33 PM Ending trivia...
09/11/2023 01:50:33 PM Setting trivia to inactive based on active_session...
09/11/2023 01:50:33 PM Starting session...
09/11/2023 01:50:33 PM Beging setting up Session...
09/11/2023 01:50:33 PM Setting up session constants...
09/11/2023 01:50:33 PM Loading config...
09/11/2023 01:50:33 PM    file_name            triviaset.csv
   question_count       1
   hint_time1           30
   hint_time2           45
   skip_time            90
   question_delay       5
   question_bonusvalue  3
   mode                 single
   music_mode           False
   order                random
   category_distribution random
   length               finite
   admins               
   output               True
   csv_encoding         utf-8

09/11/2023 01:50:33 PM Loading trivia data...
09/11/2023 01:50:33 PM Creating session set data. Population 2, session 1
09/11/2023 01:50:33 PM Finished setting up Session.
09/11/2023 01:50:33 PM Active
09/11/2023 01:50:37 PM Error on session None

**Comment by digital420** [comment]
It appears it is using the headers as an actual question. I adjusted the question count and it started with my first question and worked correctly then it went back to using the header as a question.  That first row would need to be excluded from the questions files and it should work properly at that point.

**Comment by cleartonic** [comment]
You're right - I tried making a new release that incorporated a fix for this. The versions got messed up, and I fixed the previous 2.2.2 issue. Try the latest release.
